pkg/utils/files.go

```diff
@@ -1,9 +1,12 @@
 package utils
 
 import (
+	"fmt"
 	"github.com/bugsnag/bugsnag-cli/pkg/log"
```

Comment:
It might be best for the utils package not to do any logging - leave that for the higher level functions to describe what's happening in context. (None of the other utils functions do this.)